modules/configuring-default-seccomp-profile.adoc

```diff
@@ -1,15 +1,75 @@
 [id="configuring-default-seccomp-profile_{context}"]
-= Configuring the default seccomp profile
-OpenShift ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by setting `RuntimeDefault` as following:
+= Enabling the default seccomp profile for all pods
 
-.Example
+{product-title} ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by creating a custom Security Context Constraint (SCC), editing this and adding the default seccomp profile to this custom SCC. Finally add the custom SCC to the workload's service account.
 
+[NOTE]
+====
+There is a requirement to create a custom SCC because editing the default SCCs can lead to issues when some of the platform pods deploy or {product-title} is upgraded. For more information, see the section entitled "Default security context constraints". 
```

Comment:
Break first sentence into two, with the following replacing the existing text:
"Do not edit the default SCC. Editing the default SCC can lead..."